### PR TITLE
Increase translation coverage

### PR DESCRIPTION
**Pull request by qurbat**
Added gettext (`_`) and `{% trans %}` tags where missing, and substituted improperly used single/double quotes

### CODE REVIEW

File: imgee/templates/create_label.html.jinja2 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {% block maincontent %}
   <div class="mui-panel">
-    <p class="mui--text-title mui--text-bold">{{ faicon(icon='tag', icon_size='subhead', baseline=true) }}&nbsp;&nbsp;Create label</p>
+    <p class="mui--text-title mui--text-bold">{{ faicon(icon='tag', icon_size='subhead', baseline=true) }}&nbsp;&nbsp;{% trans %}Create label {% endtrans %}</p>
```
Comment (jace) [inline imgee/templates/create_label.html.jinja2:7]: Remove the trailing space

File: imgee/templates/delete.html.jinja2 [hunk 2]
```diff
@@ -9,9 +9,9 @@
   {% endblock %}
   <div class="alert alert--error">
     <p class="mui--text-title">{{ faicon(icon='trash-alt', icon_size='subhead', baseline=true) }} Confirm delete</p><hr/>
-    <p class="mui--text-subhead">Are you sure you want to delete "<strong>{{file.title}}</strong>"?</p>
+    <p class="mui--text-subhead">{% trans file=file.title %}Are you sure you want to delete "<strong>{{ file }}</strong>{% endtrans %}"?</p>
```
Comment (jace) [inline imgee/templates/delete.html.jinja2:12]: Closing double quote must be part of the translation.

File: imgee/templates/delete.html.jinja2 [hunk 3]
```diff
@@ -9,9 +9,9 @@
...
     <p id="warning">
-      {{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>Warning: If you delete it, you will lose all the links wherever this image appears.</strong>
+      {{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>{% trans %}Warning: If you delete it, you will lose all the links wherever this image appears.{% endtrans %}</strong>
```
Comment (jace) [inline imgee/templates/delete.html.jinja2:14]: Needs better text here: "Warning: pages that use this image will need to be updated."

File: imgee/templates/delete_label.html.jinja2 [hunk 4]
```diff
@@ -10,8 +10,8 @@
   <div class="alert alert--error">
     <p class="mui--text-title">{{ faicon(icon='trash-alt', icon_size='subhead', baseline=true) }} Confirm delete</p>
     <hr/>
-    <p class="mui--text-subhead">Are you sure you want to delete the label "{{ label.title }}"?<br/></p>
-    <p>{{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>Info: Even if you delete it, the files associated with the label will NOT be deleted.</strong></p>
+    <p class="mui--text-subhead">{% trans label=label.title %}Are you sure you want to delete the label "{{ label }}{% endtrans %}"?<br/></p>
```
Comment (jace) [inline imgee/templates/delete_label.html.jinja2:13]: Closing double quote.

File: imgee/templates/delete_label.html.jinja2 [hunk 5]
```diff
@@ -10,8 +10,8 @@
...
+    <p>{{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>{% trans %}Info: Even if you delete it, the files associated with the label will NOT be deleted.{% endtrans %}</strong></p>
```
Comment (jace) [inline imgee/templates/delete_label.html.jinja2:14]: Change to: Removing this label will dissociate it from images, but will not remove those images.

File: imgee/templates/form.html.jinja2 [hunk 6]
```diff
@@ -1,11 +1,11 @@
 {% extends "layout.html.jinja2" %}
 {% from 'baseframe/forms.html.jinja2' import renderform %}
 {% from "baseframe/components.html.jinja2" import faicon %}
-{% block title %}New image{% endblock %}
+{% block title %}{% trans %}New image{% endtrans %}{% endblock %}
 
 {% block maincontent %}
 	<div class="mui-panel">
-		<p class="mui--text-title mui--text-bold">{{ faicon(icon='upload', icon_size='subhead', baseline=true) }}&nbsp;&nbsp;Upload Image</p>
+		<p class="mui--text-title mui--text-bold">{{ faicon(icon='upload', icon_size='subhead', baseline=true) }}&nbsp;&nbsp;{% trans %}Upload Image{% endtrans %}</p>
```
Comment (jace) [inline imgee/templates/form.html.jinja2:8]: Sentence case: Upload image

@vidya-ram this template has 2x `&nbsp;` between icon and text, but the delete template above has only one. This needs revising.

File: imgee/templates/profile.html.jinja2 [hunk 7]
```diff
@@ -99,9 +99,9 @@
       <div class="mui-panel">
         <p class="mui--text-subhead zero-bottom-margin">
         {% if unlabelled %}
-          Hurray! Not a single image is without a label.
+          {% trans %}Hurray! Not a single image is without a label.{% endtrans %}
```
Comment (jace) [inline imgee/templates/profile.html.jinja2:102]: Change to: There are no unlabeled images.

File: imgee/templates/profile.html.jinja2 [hunk 8]
```diff
@@ -99,9 +99,9 @@
...
         {% else %}
-          There are no images uploaded yet. <a href="{{url_for('upload_file', profile=profile.name)}}">Upload them.</a>
+          {% trans %}There are no images uploaded yet.{% endtrans %} <a href="{{url_for('upload_file', profile=profile.name)}}">{% trans %}Upload them.{% endtrans %}</a>
```
Comment (jace) [inline imgee/templates/profile.html.jinja2:104]: Remove the upload link and change text to: No images

File: imgee/templates/show_label.html.jinja2 [hunk 9]
```diff
@@ -4,9 +4,9 @@
 {% block maincontent %}
   <section class="show-label">
     <div class="mui--clearfix">
-      <div class="mui--pull-left"><span class="mui-text--title">Images with label</span> <span class="editable_label mui--d-inlineblock" id="{{label.id}}">{{label.name}}</span></div>
+      <div class="mui--pull-left"><span class="mui-text--title">{% trans %}Images with label{% endtrans %}</span> <span class="editable_label mui--d-inlineblock" id="{{label.id}}">{{label.name}}</span></div>
```
Comment (jace) [inline imgee/templates/show_label.html.jinja2:7]: This is an example of the label being part of a sentence, and yet being (necessarily) outside the trans tag. The correct fix is to remove the text and use an icon representing a label. @vidya-ram you'll have to pick one.

File: imgee/templates/show_label.html.jinja2 [hunk 10]
```diff
@@ -4,9 +4,9 @@
...
       {% if current_auth.user %}
-        <a class="mui-btn mui-btn--small mui-btn--danger mui--pull-right" href="{{url_for('delete_label', profile=profile.name ,label=label.name)}}" title='Delete label "{{label.name}}"'>{{ faicon(icon='trash-alt', baseline=true) }}&nbsp;<span id="delete">Delete Label</span></a>
+        <a class="mui-btn mui-btn--small mui-btn--danger mui--pull-right" href="{{url_for('delete_label', profile=profile.name ,label=label.name)}}" title='Delete label "{{label.name}}"'>{{ faicon(icon='trash-alt', baseline=true) }}&nbsp;<span id="delete">{% trans %}Delete Label{% endtrans %}</span></a>
```
Comment (jace) [inline imgee/templates/show_label.html.jinja2:9]: Fix for sentence case.

**Review by jace — CHANGES_REQUESTED** [review]
Needs fixes for (a) some trans tags, (b) some English sentences, and (c) some HTML/CSS styling.

File: imgee/templates/delete.html.jinja2 [hunk 11]
```diff
@@ -9,9 +9,9 @@
   {% endblock %}
   <div class="alert alert--error">
     <p class="mui--text-title">{{ faicon(icon='trash-alt', icon_size='subhead', baseline=true) }} Confirm delete</p><hr/>
-    <p class="mui--text-subhead">Are you sure you want to delete "<strong>{{file.title}}</strong>"?</p>
+    <p class="mui--text-subhead">{% trans file=file.title %}Are you sure you want to delete "<strong>{{ file }}</strong>"?{% endtrans %}</p>
```
Comment (vidya-ram) [inline imgee/templates/delete.html.jinja2:12]: Do we need the strong tag here? Quotes would suffice, no?

File: imgee/templates/index.html.jinja2 [hunk 12]
```diff
@@ -12,12 +12,12 @@
         <a href="{{ url_for('index') }}"><img src="{{ url_for('static', filename='img/logo.png') }}"></a>
       </div>
       <div>
-        <p class="mui--text-title">Hasgeek’s image library</p><hr />
+        <p class="mui--text-title">{% trans %}Hasgeek’s image library{% endtrans %}</p><hr />
         {% if current_auth.user %}
           <p class="mui--text-subhead">Welcome <strong>{{ current_auth.user.fullname }}!</strong></p>
```
Comment (vidya-ram) [inline imgee/templates/index.html.jinja2:17]: trans tag missing

File: imgee/templates/show_label.html.jinja2 (hunk 9 above)
Comment (vidya-ram) [reply imgee/templates/show_label.html.jinja2:7]: The labels are user defined and font awesome might not have options available for all and naming conventions will be different too.

Comment (vidya-ram) [reply imgee/templates/show_label.html.jinja2:7]: The span with `editable_label` class has a JS function attached to it as well. It opens form to edit the label on click

File: imgee/templates/form.html.jinja2 (hunk 6 above)
Comment (vidya-ram) [reply imgee/templates/form.html.jinja2]: Add css class to include margin

**Pull request merged**